packages/clerk-js/src/ui/localization/defaultEnglishResource.ts

```diff
@@ -426,16 +427,55 @@ export const defaultResource: DeepRequired<LocalizationResource> = {
     start: {
       headerTitle__members: 'Members',
       headerTitle__settings: 'Settings',
+      headerSubtitle__members: 'View and manage organization members',
+      headerSubtitle__settings: 'View and manage organization members',
     },
     profilePage: {
+      title: 'Organization Profile',
+      subtitle: 'Manage organization profile',
       successMessage: 'The organization has been updated.',
-      leave: {
-        title: 'Leave organization',
-        messageLine1:
-          'Are you sure you want to leave this organization? You will lose access to this organization and its applications.',
-        messageLine2: 'This action is permanent and irreversible.',
-        successMessage: 'You have left the organization.',
+      dangerSection: {
+        title: 'Danger',
+        leave: {
```

Comment:
Nitpicking, but lets rename to `leaveOrganization` to make it a little more explicit